### PR TITLE
fix: [2.5] Check if the dynamic fields contain any static fields

### PR DESCRIPTION
**Pull request by xiaocai2333**
issue: #38024 

master pr: #38025

### CODE REVIEW

File: internal/proxy/util.go [hunk 1]
```diff
@@ -1792,11 +1792,20 @@ func verifyDynamicFieldData(schema *schemapb.CollectionSchema, insertMsg *msgstr
 			for _, rowData := range field.GetScalars().GetJsonData().GetData() {
 				jsonData := make(map[string]interface{})
 				if err := json.Unmarshal(rowData, &jsonData); err != nil {
-					return err
+					log.Warn("insert invalid dynamic data, milvus only support json map",
```
Comment (zhengbuqian) [inline internal/proxy/util.go:1795]: incorrect user input should not cause a warn log? warn log is more like for internal system warns.

comment is not blocking, not requesting a change in this PR

**Review by zhengbuqian — COMMENTED** [review]
/lgtm

**Comment by xiaocai2333** [comment]
close with https://github.com/milvus-io/milvus/pull/38025
/close

**Pull request closed without merging**